loops-and-arrays.js

```diff
@@ -37,6 +53,13 @@ const numbers = [2, 4, 5, 29, 38];
 
 const squares = [30, 2, 8, 24, 11];
 
+console.log(squares);
+
+for (let i = 0; i < squares.length; i++) {
+   squares[i] = Math.pow(squares[i],3)
```

Comment:
nice, deze methode kende ik nog niet. Ik heb zelf gewoon `square * square * square` gedaan ;) Maar dit is natuurlijk veel mooier.